docs/conf.py

```diff
@@ -12,6 +12,7 @@
 # serve to show the default.
 
 from email import message_from_string
+import mock
```

Comment:
I don't know that `mock` will be importable:  we don't have it as a dependency (and don't want it, particularly just for this case).

Per the [RTD FAQ entry](http://read-the-docs.readthedocs.io/en/latest/faq.html#i-get-import-errors-on-libraries-that-depend-on-c-modules), we are going to need to suppress adding the `GRPC_EXTRAS` dependencies to `REQUIREMENTS` on RTD:  something like:

```python
if sys.version_info[:2] == (2, 7) and 'READTHEDOCS' not in os.environ:
    REQUIREMENTS.extend(GRPC_EXTRAS)
```

This change to `setup.py` might actually mean we don't need to change `docs/conf.py` at all.